crawlers/mooncrawl/mooncrawl/nft/cli.py

```diff
@@ -95,31 +99,7 @@ def get_latest_summary_block() -> Optional[int]:
 
 
 def get_latest_nft_labeled_block(db_session: Session) -> Optional[int]:
```

Comment:
@Yhtiyar : Put a `TODO` here describing that this needs to be changed before `synchronize` command works.